_includes/configuration.md

```diff
@@ -60,8 +57,7 @@ database.
 The `postgresql.conf` file used with {{ site.data.product.title_short }} databases requires
 specific settings for correct operation. For example, it must correctly
 reclaim table space, control session timeouts, and format the PostgreSQL
-server log for improved system support. Due to these requirements, Red
-Hat recommends that external {{ site.data.product.title_short }} databases use a
+server log for improved system support. It is recommended that external {{ site.data.product.title_short }} databases use a
```

Comment:
I think we can remove the `title_short` reference completely for simplicity

Reply:
@bdunne  All the instances has been replaced with ManageIQ.

Reply:
@Rajitha20H Switching to ManageIQ is not what @bdunne was suggesting.  Those values were set to `{{ site.data.product.title_short }}` because we they need to be changed between upstream and downstream, so ideally we shouldn't use the name `ManageIQ` anywhere in the docs, but instead use `{{ site.data.product.title_short }}`

What @bdunne was suggesting was that in this particular instance, having the product name doesn't provide any more information to the reader, so we can remove the product name entirely.